client/components/TestQueue/Conflicts/AssertionConflictsTable.jsx

```diff
@@ -4,13 +4,23 @@ import { UserPropType } from '../../common/proptypes';
 import PropTypes from 'prop-types';
 
 const AssertionConflictsTable = ({ conflictingResults, testers }) => {
+  const commandString = scenario => {
+    return `Assertions for 'After ${scenario.commands
+      .map(command => command.text)
+      .join(', then ')}'`;
```

Comment:
```suggestion
      .join(' then ')}'`;
```

Since introducing the v2 test format, the CG has wanted to standardize on [`<space>then<space>` rather than `,<space>then<space>`](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#commandsjson-processing-building-strings-for-command-display) for these command concatenations, so we can avoid that where possible.